src/opendistro-es/values.yaml

```diff
@@ -1,10 +1,8 @@
 kibana:
   enabled: true
   image: amazon/opendistro-for-elasticsearch-kibana
-  #image: ehealthafrica/flask
-  imageTag: 1.1.0
-  #imageTag: latest
-  replicas: 1
+  imageTag: 1.2.0
```

Comment:
not 1.2.1?

Reply:
this should actually be `1.2.1` will update tomorrow